test/e2e/framework/framework.go

```diff
@@ -177,76 +181,37 @@ func (f *Framework) GetService(cluster framework.ClusterIndex, name, namespace s
 	return framework.KubeClients[cluster].CoreV1().Services(namespace).Get(context.TODO(), name, metav1.GetOptions{})
 }
 
-func (f *Framework) AwaitServiceImportIP(srcCluster, targetCluster framework.ClusterIndex, svc *v1.Service) *mcsv1a1.ServiceImport {
-	serviceIP := f.GetServiceIP(srcCluster, svc, false)
-
-	return f.AwaitServiceImportWithIP(targetCluster, svc, serviceIP)
-}
+func (f *Framework) AwaitAggregatedServiceImport(targetCluster framework.ClusterIndex, svc *v1.Service, clusterCount int) {
+	By(fmt.Sprintf("Retrieving ServiceImport for %q in ns %q on %q", svc.Name, svc.Namespace,
```

Comment:
Does this function check if service import is present? or we want to check if the status have all the clusters which exported the service?

Reply:
It checks if an aggregated `ServiceImport` is present and its status contains the expected cluster count. This replaces the prior `AwaitServiceImportCount` function.